controller/array_action/array_mediator_interface.py

```diff
@@ -243,13 +243,13 @@ def get_object_by_id(self, object_id, object_type):
         raise NotImplementedError
 
     @abstractmethod
-    def create_snapshot(self, name, volume_name, pool_id=None):
+    def create_snapshot(self, volume_id, snapshot_name, pool):
         """
         This function should create a snapshot from volume in the storage system.
         Args:
-            name           : name of the snapshot to be created in the storage system
-            volume_name    : name of the volume to be created from
-            pool_id: pool_id of the volume to find the snapshot more efficiently.
+            volume_id    : id of the volume to be created from
+            snapshot_name           : name of the snapshot to be created in the storage system
```

Comment:
why so much indentation before the colon?

Reply:
good question, it was automatic by the refactor... I will fix

Reply:
done